### ISSUE TITLE
Support <time> for article publish date/time in custom front matter

### ISSUE THREAD

**Issue by keikhcheung** [deathau/markdownload#61]
Articles online often contains `<time datetime="…">` for their published date & time.
It would be great if it can be added as an option in custom front matter.

**Comment by xuguangyue** [comment]
Yes, want this feature too!

**Comment by asapvw** [comment]
Yes, it would be great to include `datePublished` which is now supported in Readability.js.
